[EDU-1665] - Chat rooms and messages example (#2288)
ably/docs: examples/chat-rooms-messages/page.md

# Joining chat rooms and sending messages

Use rooms and messages to enable users to join rooms, send and receive messages.

Joining chat rooms and sending messages are core features of many applications with chat functionalities. They enable users to connect with others and communicate in real-time.

Chat rooms and messaging add value in different applications. For example, in customer support platforms, agents and clients can join dedicated rooms to discuss issues and share information. In larger applications, such as social networking or team collaboration tools, users can join multiple rooms based on topics or projects, fostering engagement and facilitating group discussions.

> Rooms and messages can be implemented using [Ably Chat](https://ably.com/docs/products/chat). The Chat SDK contains a set of purpose-built APIs that abstract away the complexities involved in architecting chat features. It is built on top of Ably's core platform, and so it provides the same performance and scaling guarantees.

Reviewer: ```suggestion
Rooms and messages are implemented using [Ably Chat](https://ably.com/docs/products/chat). The Chat SDK contains a set of purpose-built APIs that abstract away the complexities involved in architecting chat features. It is built on top of Ably's core platform, and so it provides the same performance and scaling guarantees.
```
